examples/automated-testing-performance-tests.md

```diff
@@ -0,0 +1,118 @@
+# Performance Testing - Examples
```

Comment:
@bubafinder can we link this page somewhere? I suggest we put the link somewhere here https://github.com/ExtensionEngine/tech-guide/blob/master/recipes/automated-testing.md#performance-tests